### ISSUE TITLE
Ability to select PTZ preset without trigger

### ISSUE THREAD

**Issue by rjsears** [flyjmz/jmzSmartThings#7]
Hello, will your app allow me to simply select a PTZ preset. I don't need to trigger recording, etc I would just like to use ActionTiles in combination with WebCore and smartthings to move my camera to set positions.

Thanks

**Comment by flyjmz** [comment]
Yes, you could set it up and not set up triggers or anything else.  The
apps and device handlers allow each camera to be a device on smartthings
that you can treat like any other device.  So you could use WebCORE or
something to send PTZ positions to the camera directly based on whatever
you want.
